src/utils/exerciseServer.ts

```diff
@@ -311,6 +314,30 @@ async function exerciseServerWorker(testDir: string, tsserverPath: string, repla
                                 ],
                             }
                         });
+
+                        // Auto-import completions frequently cause issues with completion entry details.
+                        if (requestWithAutoImports) {
+                            const completionEntries = invokedResponse.body.entries;
+                            for (let entry of completionEntries) {
+                                if (entry.hasAction && entry.source !== undefined && "data" in entry) {
+                                    const { name, source, data } = entry;
+                                    
+                                    // 'completionEntryDetails' can take multiple entries; however, it's not useful for diagnostics
```

Comment:
Curious to hear if others think this is bad - this is pretty likely to be slower, but I like to look at the last command and know exactly what went wrong.

I could make requests on 5 at a time in the future.

Reply:
AFAIK nobody uses the multiple-details-at-a-time command. IIRC it’s not optimized whatsoever; it just loops at the super top level and doesn’t cache anything.